src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -7,6 +7,7 @@
 using System.ComponentModel.Design;
 using System.Diagnostics;
 using System.Drawing;
+using System.Drawing.Imaging;
```

Comment:
This using needs to be if def as well, that is why the build is currently failing for UAP.